gammapy/modeling/models/temporal.py

```diff
@@ -189,6 +190,30 @@ def integral(self, t_min, t_max):
         t_ref = Time(pars["t_ref"].quantity, format="mjd")
         value = self.evaluate(t_max, t0, t_ref) - self.evaluate(t_min, t0, t_ref)
         return -t0 * value / self.time_sum(t_min, t_max)
+        
+    def sample_time(self, n_events, random_state=0):
```

Comment:
Shouldn't this method have `tmin` and `tmax` arguments to follow the logic of the other `TemporalModels`?

Reply:
Yes, good idea. However, it seems that `numpy.random.Generator.exponential` does not allow to sample into a given `tmin`-`tmax` interval. Any suggestion?